packages/server/src/controllers/bookings.ts

```diff
@@ -1,15 +1,17 @@
 import { Context } from "koa";
 import {
-  getBookingById,
-  getAllBookings,
+  retrieveBookingById,
+  retrieveAllBookings,
   updateBookingById,
   BookingStatus,
+  retrieveBookingsByInterval,
 } from "../db/bookings";
 import {
   validateNewBookingData,
   createNewBooking,
   haveBookingDatesChanged,
 } from "../utils/booking";
+import { format, toDate } from "date-fns";
```

Comment:
To date is unused
```suggestion
import { format } from "date-fns";
```

Reply:
run yarn install on your project root to instal eslint.
You should also install the eslint extension for vscode to get errors directly in your editor

Reply:
Thanks!